### PR TITLE
CE-2044: Adding support for AzureSQL

### PR DESCRIPTION
**Pull request by marutinandanpandya**

### CODE REVIEW

File: config-sample.yml [hunk 1]
```diff
@@ -9,7 +9,6 @@ copperegg:
   - MS_NET_CLR
   - MS_ASP_NET
   - MS_Web_Services
-  - MS_MSSQL
```
Comment (Neelakshi1808) [inline config-sample.yml:12]: Why are we making changes to this sample file ? If changes are made in the way config works, lets reflect in sample file as well.

File: CopperEgg.psm1 [hunk 2]
```diff
@@ -106,7 +106,7 @@ param(
     $data
   )
   $rtn = ""|select Response,Return
```
Comment (Neelakshi1808) [inline CopperEgg.psm1:108]: While making final commit following things needs to be done :
 Remove config.yml file
 Include MSSQL again in config-sample.yml
 Include MSSQL block again in  Start-CopperEggMonitor.ps1
 Change URI to point to api.copperegg.com instaed of api.staging.cuegg.net

Comment (marutinandanpandya) [inline CopperEgg.psm1:108]: Included MSSQL again in config-sample.yml
Included MSSQL block again in Start-CopperEggMonitor.ps1
Changed URI to point to api.copperegg.com instaed of api.staging.cuegg.net
removed config.yml

File: Initialize-Dashboards.ps1 [hunk 3]
```diff
@@ -555,6 +556,212 @@ function Initialize-Dashboards {
...
+                  "style"="values";
+                  "match"="all";
+                  "metric" = @($gname, "12", $warray[11].name);
+                   };
+            "13" = @{
+                  "type"="timeline";
+                  "style"="values";
+                  "match"="all";
+                  "metric" = @($gname, "13", $warray[11].name);
+                   };
```
Comment (Neelakshi1808) [inline Initialize-Dashboards.ps1:656]: From here onwards metric is same.

File: Start-CopperEggJob.ps1 [hunk 4]
```diff
@@ -84,7 +85,91 @@ param(
...
+          $metric_data.Set_Item($CounterName, $sample.cntr_value)
+        }
+
+        $apicmd = '/revealmetrics/samples/' + $group_name + '.json'
+        $EpochSecs=[int][double]::Parse($(Get-Date -date (Get-Date).ToUniversalTime()-uformat %s))
+        $payload = @{
+          identifier             = [string]$hostname;
+          timestamp              = $EpochSecs
+          values                 = @{
+            'Cache_Hit_Ratio_Base' = $metric_data['Cache Hit Ratio Base']
```
Comment (Neelakshi1808) [inline Start-CopperEggJob.ps1]: There are 30 metrics, why are we assigning only 29 values ?

File: Start-DebugMonitor.ps1 [hunk 5]
```diff
@@ -97,6 +97,91 @@ function Start-DebugMonitor {
...
+
+                Foreach ($sample in  $samples) {
+                  $metric_data.Set_Item($sample.counter_name.trim(), $sample.cntr_value)
+                }
+                $EpochSecs=[int][double]::Parse($(Get-Date -date (Get-Date).ToUniversalTime()-uformat %s))
+                $payload = @{
+                  identifier             = [string]$h;
+                  timestamp              = $EpochSecs
+                  values                 = @{
+                    'Cache_Hit_Ratio_Base' = $metric_data['Cache Hit Ratio Base']
```
Comment (Neelakshi1808) [inline Start-DebugMonitor.ps1]: 30 metrics, assigning only 29 values.

**Pull request closed without merging**